test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eux
+
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+test-build() {
+  nix-build -A "$1"
+  for aci in "$ROOT_DIR/result/"*.aci; do
```

Comment:
I'm thinking that the bash glob will work equally when actool is used without the loop. What am I missing?